packages/formation-react/src/components/Telephone/Telephone.mdx

```diff
@@ -122,32 +119,33 @@ import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/f
+| Key                             | Number       | Description                                                                                                  |
+| :------------------------------ | :----------- | :----------------------------------------------------------------------------------------------------------- |
+| `222_VETS`                      | `8772228387` | VA Help Line                                                                                                 |
+| `4AID_VET`                      | `8774243838` | National Call Center for Homeless Veterans                                                                   |
+| `711`                           | `711`        | Telecommunications Relay Service                                                                             |
+| `911`                           | `911`        | Emergency!                                                                                                   |
+| `CAREGIVER`                     | `8552603274` | VA National Caregiver Support Line                                                                           |
+| `CRISIS_LINE`                   | `8002738255` | Veterans Crisis hotline                                                                                      |
+| `CRISIS_TTY`                    | `8007994889` | Veterans Crisis hotline TTY                                                                                  |
+| `DMC`                           | `8008270648` | Debt Management Center                                                                                       |
+| `DMC_OVERSEAS`                  | `6127136415` | Debt Management Center (Overseas)                                                                            |
+| `DMDC_DEERS`                    | `8663632883` | Defense Manpower Data Center (DMDC) + Defense Enrollment Eligibility Reporting System (DEERS) Support Office |
+| `DS_LOGON_TTY`                  | `8663632883` | Defense Manpower Data Center TTY                                                                             |
+| `DS_LOGON`                      | `8005389552` | Defense Manpower Data Center                                                                                 |
+| `FEDERAL_RELAY_SERVICE`         | `8008778339` | Federal Relay Service                                                                                        |
+| `GI_BILL`                       | `8884424551` | Education Call Center (1-888-GI-BILL-1)                                                                      |
+| `GO_DIRECT`                     | `8003331795` | Go Direct/Direct Express (Treasury)                                                                          |
+| `HELP_DESK`                     | `8446982311` | VA Help desk                                                                                                 |
```

Comment:
I don't think this is right... Anyway we already have the `800-698-2411` help desk number under `VA_311` (maybe we should make this an alias under `VA_411`?

> The VA311 phone number is changing from 844-698-2311 to 800-698-2411.

See https://dsva.slack.com/archives/CLY6Q69RV/p1603385244128600

Whatever the case, this table needs to match the `contact` list - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/contacts.js#L46 - maybe we should also think about automating this?

Reply:
There are several numbers that have duplicated, I just merged master had 0 lines changed then edited two numbers... hard to track with this formatting really need to get linter/formatting on this repo.

Let me go cross check with master branch

Reply:
So here is that first number you referenced which stayed the same [VA_411](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/Telephone.mdx#L149).

Also just tried to call that old HELP_DESK phone number and it rings then gets immediate hung up... let me go find my change where I updated this phone number in `formation-react` a couple months ago.

Reply:
@Mottie here is the [PR](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/437) that this doc update was supposed to reflect way back when.

Reply:
Ok looks good! Thanks!

Reply:
Cross referencing slack convo for future/documentation clarity 

https://dsva.slack.com/archives/CBU0KDSB1/p1610393236325800

Reply:
@bkjohnson This `HELP_DESK` number should have been changed to `800-698-2411`.